request_handlers.cc

```diff
@@ -212,56 +212,59 @@ RequestHandler::Status ReverseProxyRequestHandler::handleRequest(const Request&
   boost::system::error_code ec;
 
   // Attempt to query the host using the HTTP protocol
-  std::string port = "http"; 
+  std::string port = "http";
   if (reverse_proxy_port != "") {
     port = reverse_proxy_port;
   }
-  std::string modified_proxy_host = 
-  ReverseProxyRequestHandler::sanitizeProxyHost(reverse_proxy_host); 
-  
+  std::string modified_proxy_host =
+  ReverseProxyRequestHandler::sanitizeProxyHost(reverse_proxy_host);
+
+  // Avoid "host not found" error
```

Comment:
I think this comment might be outdated since we aren't using the `boost::asio::ip::resolver_query_base::numeric_service` thing anymore